hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieFileGroupReaderBasedRecordReader.java

```diff
@@ -206,10 +232,10 @@ private static String getLatestCommitTime(InputSplit split, HoodieTableMetaClien
   }
 
   /**
-   * Convert FileSplit to FileSlice, but save the locations in 'hosts' because that data is otherwise lost.
+   * Convert FileSplit to FileSlice
    */
-  private static FileSlice getFileSliceFromSplit(FileSplit split, Map<String, String[]> hosts, FileSystem fs, String tableBasePath) throws IOException {
```

Comment:
got rid of hosts, will create a followup pr to allow this info and other engine specific file things to get passed naturally through the filegroup structure